### ISSUE TITLE
Possible to Zoom to a div with an ID on Click

### ISSUE THREAD

**Issue by sikandarchishty** [aarondail/react-zoomable-ui#42]
Is it possible to Zoom to a specific div with any Selector like ID or Class name on button click?

**Comment by aarondail** [comment]
OK I added a new method on the `ViewPortCamera` to support this and make it marginally easier and better: https://github.com/aarondail/react-zoomable-ui/blob/master/docs/api/classes/viewportcamera.md#centerfitelementintoview 

I released it in `0.11.0` just now.

This new method doesn't really save you a ton of code, you still have to have add some code that gets the `ViewPortCamera` like above, but I added some options that might be useful... and in the process of adding this I also fixed a few other bugs I hadn't noticed before and made the zooming animation feel better. So thanks for opening this. Lmk if you have a chance to check it out and see if it works for you.

In any case, code like this should work now:
```
// Assume spaceRef is a React ref to a <Space> component
const div = document.getElementById("my-div-id")
spaceRef.current.viewPort.camera.centerFitElementIntoView(div);
```

I also added a new button to [the overview example](https://aarondail.github.io/react-zoomable-ui/example/#/overview) to demo the behavior.
